src/util/PlatformEx.java

```diff
@@ -0,0 +1,32 @@
+package util;
+
+import javafx.application.Platform;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import service.ServiceManager;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class PlatformEx {
```

Comment:
A brief comment about the purpose of the class can be helpful. The class name is not self-explanatory.